src/vs/workbench/parts/tasks/electron-browser/taskPanel.ts

```diff
@@ -0,0 +1,133 @@
+import { IThemeService } from 'vs/platform/theme/common/themeService';
+import { ITaskService } from 'vs/workbench/parts/tasks/common/taskService';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { TPromise } from 'vs/base/common/winjs.base';
+import { ICommandService } from 'vs/platform/commands/common/commands';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+import { domElement } from 'vs/workbench/parts/tasks/electron-browser/taskButtons';
+import { buttonBackground, buttonForeground, textLinkForeground, selectBackground } from 'vs/platform/theme/common/colorRegistry';
+import { IStorageService, StorageScope } from 'vs/platform/storage/common/storage';
+import { IOpenerService } from 'vs/platform/opener/common/opener';
+
+const TASK_PANEL_ID = 'workbench.panel.task';
+
+export class TaskPanel extends Panel {
+
+	private _actions: IAction[];
+	private taskExperimentPart5 = 'workbench.tasks.feedbackAnswered';
+
+	constructor(
+
+		@ITelemetryService telemetryService: ITelemetryService,
+		@IThemeService protected themeService: IThemeService,
+		@ITaskService private taskService: ITaskService,
+		@IInstantiationService private _instantiationService: IInstantiationService,
+		@ICommandService private commandService: ICommandService,
+		@IStorageService private storageService: IStorageService,
+		@IOpenerService private openerService: IOpenerService,
+
+	) {
+		super(TASK_PANEL_ID, telemetryService, themeService);
+	}
+
+	public create(parent: Builder): TPromise<any> {
+		super.create(parent);
+		dom.addClass(parent.getHTMLElement(), 'task-panel');
+
+		let builder = parent.innerHtml(domElement());
+		let buttons = builder.select('.mockup-button');
+		let links = builder.select('.linkstyle');
+		let taskItems = builder.select('.task-item');
+		let yesButton = builder.select('.yes-telemetry');
+		let noButton = builder.select('.no-telemetry');
+		let githubLink = builder.select('.linkstyle');
+		let clickFeedback = builder.select('.header-item');
+		let thanks = builder.select('.thanks');
+
+		buttons.style('background-color', this.themeService.getTheme().getColor(buttonBackground).toString());
+		buttons.style('color', this.themeService.getTheme().getColor(buttonForeground).toString());
+		links.style('color', this.themeService.getTheme().getColor(textLinkForeground).toString());
+		taskItems.style('background-color', this.themeService.getTheme().getColor(selectBackground).toString());
+
+		this.themeService.onThemeChange(() => {
+			buttons.style('background-color', this.themeService.getTheme().getColor(buttonBackground).toString());
+			buttons.style('color', this.themeService.getTheme().getColor(buttonForeground).toString());
+			links.style('color', this.themeService.getTheme().getColor(textLinkForeground).toString());
+			taskItems.style('background-color', this.themeService.getTheme().getColor(selectBackground).toString());
+		});
```

Comment:
Don't duplicate all this, have some function that's attached to onThemeChange and also call it manually during `create()`